### PR TITLE
Implement /createRoom power_level_content_override

### PR DESCRIPTION
**Pull request by t3chguy**
Doesn't look like sytest covered this parameter
It is one used by spaces so would be nice to land swiftly, lmk if you need me to do more.

### CODE REVIEW

File: clientapi/routing/createroom.go [hunk 1]
```diff
@@ -38,16 +38,17 @@ import (
...
+	Name                      string                        `json:"name"`
+	Visibility                string                        `json:"visibility"`
+	Topic                     string                        `json:"topic"`
+	Preset                    string                        `json:"preset"`
+	CreationContent           map[string]interface{}        `json:"creation_content"`
+	InitialState              []fledglingEvent              `json:"initial_state"`
+	RoomAliasName             string                        `json:"room_alias_name"`
+	GuestCanJoin              bool                          `json:"guest_can_join"`
+	RoomVersion               gomatrixserverlib.RoomVersion `json:"room_version"`
+	PowerLevelContentOverride map[string]interface{}        `json:"power_level_content_override"`
```
Comment (neilalexander) [inline clientapi/routing/createroom.go:51]: You can probably simplify this by just setting the type here to `json.RawMessage` to defer un-marshalling. Then you won’t need to marshal it again before un-marshalling it over the top of `&powerLevelContent`.

Comment (t3chguy) [reply clientapi/routing/createroom.go]: Great; much cleaner. Done.

**Comment by neilalexander** [comment]
Sytest meltdown in CI seems to be related to matrix-org/sytest#1011.

**Review by neilalexander — APPROVED** [review]
LGTM pending Sytest (which is actually running now).

**Pull request merged**